activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/validation/extensions/ExtensionsModelValidator.java

```diff
@@ -1,171 +1,50 @@
 /*
- * Copyright 2019 Alfresco, Inc. and/or its affiliates.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Copyright 2005-2018 Alfresco Software, Ltd. All rights reserved.
```

Comment:
restore community header, please check if that happened in other files too

Reply:
Restored, and checked that it didn't happen anywhere else